opennlp-tools/src/main/java/opennlp/tools/ml/maxent/GISModel.java

```diff
@@ -78,6 +96,23 @@ public GISModel(Context[] params, String[] predLabels, String[] outcomeNames,
   }
 
   /**
+   * Creates a new model with the specified parameters, outcome names, and
+   * predicate/feature labels.
+   *
+   * @param params
+   *          The parameters of the model.
+   * @param predLabels
+   *          The names of the predicates used in this model.
+   * @param outcomeNames
+   *          The names of the outcomes this model predicts.
+   * @param prior
+   *          The prior to be used with this model.
+   */
+  public GISModel(Context[] params, String[] predLabels, String[] outcomeNames, Prior prior) {
+    this(params, predLabels, outcomeNames, 1, 0, prior);
+  }
+
```

Comment:
We need to also deprecate that method in AbstractModel, just did that and updated this PR.